gogs_client/entities.py

```diff
@@ -13,17 +13,24 @@ def json_get(parsed_json, key):
     return parsed_json[key]
 
 
-class GogsUser(object):
+class GogsEntity(object):
+    def __init__(self, json):
+        self.json = json
+
+
+class GogsUser(GogsEntity):
     """
      An immutable representation of a Gogs user
     """
 
-    def __init__(self, user_id, username, full_name, email, avatar_url):
+    def __init__(self, user_id, username, full_name, email, avatar_url, json={}):
+        super(GogsUser, self).__init__(json=json)
         self._id = user_id
         self._username = username
         self._full_name = full_name
         self._email = email
         self._avatar_url = avatar_url
+        self._json = json
```

Comment:
What is the point of this `_json` field? It's never used, and the JSON representation is already stored by the parent `GogsEntity` class.